bebraven/platform
# Refactor our views to use two different layouts: accounts vs content editor

Instead of everything using the default application layout, now controllers
related to Content Editor features can use the 'content_editor' layout and 
controllers related to folks trying to access / use a resource (like the Portal)
with their account can use the 'accounts' layout. All the other admin stuff
still uses the default application layout.

This will allow us to more cleanly split out the shared stuff for the two different
layouts. E.g. headers and footers for account access.

Note: Rails 6 moved from the Sprockets assets pipeline to a Webpack pipeline for javascripts. This PR also removes the old Sprokects javascripts files which weren't used. See: https://prathamesh.tech/2019/08/26/understanding-webpacker-in-rails-6/

### Test Plan:
1. go to the Content Editor, load up some real content, and make sure it looks the same. Make sure it links in the correct javascript too.
2. go to the sign-in, forgot password, sign-up, etc pages and make sure they get the 
  shared headers and footers. Also check the javascript.
